Libraries/Hotcakes.Shipping.UpsFreight/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Reflection;
```

Comment:
This is missing the open source license header

Reply:
done,added license header.